### DISCUSSION TITLE
My Application for GitHub Copilot Student Pack is Pending: No Email or Accpetion

### DISCUSSION THREAD

**Discussion by Arandinglv** [community/community#143293] [GitHub Education]
### Hi everyone,

I sunmitted my application for GitHub Copilot Student Pack in October 28th, but I haven't receive any email or message about the process of the application. How can I do to manage this situation? 

> Your application cannot be reviewed until you fix the following: You have an outstanding discount request for @arandinglv. We will get back to you soon.

*By the way, I am a visit student in another school, so I'm distant from original school. But I have both certificate, student card and visit student card*.

**Comment by CemWebDev** [comment]
Hi @arandinglv,

To help resolve your GitHub Copilot Student Pack application status, I recommend trying to link your account through the **GitHub Mobile** app. Some users have reported that after verifying their accounts on GitHub Mobile, their applications were processed more quickly. You can try downloading GitHub Mobile and linking your account from there.

If this doesn't resolve the issue, I suggest reaching out directly to the GitHub Education support team. They can provide more detailed information regarding the "outstanding discount request" status on your application, which might expedite the process.

Hope your application gets sorted out soon!

**Comment by Arandinglv** [comment]
Thank you for your kind message.

Could you please provide links to the GitHub Education support team? I only
found the GItHub Education discussion to ask for help.

By the way, do I need to use the GitHub app to apply for the GitHub Copilot
Student Pack directly? Maybe I didn't understand what you meant.

Thanks again for your help.

Cem ***@***.***> 于2024年11月1日周五 22:32写道：

**Comment by CemWebDev** [comment]
@arandinglv 

 Here are some steps to help clarify:

1. **GitHub Education Support**: You can contact GitHub Education support directly by submitting a request through their support page here: [GitHub Support for Education](https://support.github.com/contact?tags=education). This link should take you to the official GitHub support contact form where you can select "GitHub Education" as the topic.

2. **GitHub Mobile App**: You don't need to apply for the GitHub Student Pack directly through the GitHub Mobile app. Instead, some users have found that verifying their account in the GitHub Mobile app helps speed up the verification process. Simply linking your account in the mobile app can sometimes make the verification easier.

If you still face any issues, the GitHub Education support team should be able to provide further assistance. Good luck, and let me know if you need any more help!
